install_config/installing-pipelines.adoc

```diff
@@ -40,11 +40,13 @@ include::modules/op-pipelines-operator-in-restricted-environment.adoc[leveloffse
 
 * You can learn more about installing Operators on {OCP} in the link:https://docs.openshift.com/container-platform/latest/operators/admin/olm-adding-operators-to-cluster.html[adding Operators to a cluster] section.
 
-* To install {tekton-chains} using the {pipelines-title} Operator, see xref:../secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc#using-tekton-chains-for-openshift-pipelines-supply-chain-security[Using {tekton-chains} for {pipelines-title} supply chain security].
+* To configure {tekton-chains}, see xref:../secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc#using-tekton-chains-for-openshift-pipelines-supply-chain-security[Using {tekton-chains} for {pipelines-title} supply chain security].
+
+* To enable and configure {tekton-results}, see xref:../records/using-tekton-results-for-openshift-pipelines-observability.adoc#using-tekton-results-for-openshift-pipelines-observability[Using {tekton-results} for {pipelines-shortname} observability].
```

Comment:
Tekton result will be enabled by default like Tekoton Chains and users can add their configuration

Reply:
okay this merits a bit of a reconfiguration of the docs